trlx/trainer/nn/ilql_models.py

```diff
@@ -33,6 +33,17 @@ def topk_mask(xs: torch.FloatTensor, k: int):
     return torch.where(xs < mintop, -np.inf * torch.ones_like(xs, dtype=xs.dtype), xs)
 
 
+def batched_index_select(x, idxs, dim):
+    """
+    x: [batch, seq, hidden]
+    idxs: [batch, idxs_seq]
+    returns: [batch, idxs_seq, hidden]
```

Comment:
Aside: We should probably remove the `torchtyping` dependency since we never use it 😄